src/popup/components/AccountCardWrapper.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+  <BtnBase
+    :class="['account-card', { selected }]"
+    :to="to"
+    :disabled="!selected"
+    :bg-color="color"
+    data-cy="account-card"
+  >
+    <div class="top">
+      <slot
+        name="top"
+        :color="color"
+      />
+    </div>
+
+    <div class="middle">
+      <slot name="middle" />
+    </div>
+
+    <slot name="bottom" />
```

Comment:
I suggest to wrap this with additional div to avoid further problems and allow to see the structure of the component in the DevTools.